pkg/servicemesh/servicemesh-member-roll.go

```diff
@@ -0,0 +1,266 @@
+package servicemesh
```

Comment:
re-name file to just member-roll.go. servicemesh name already present in package's name